packages/evolution_data_server.rb

```diff
@@ -2,32 +2,48 @@
 
 class Evolution_data_server < Package
   description 'Centralized access to appointments and contacts'
-  @_ver = '3.40.3'
+  @_ver = '3.46.0'
   version @_ver
   license 'LGPL-2 or LGPL-3, BSD and Sleepycat'
-  compatibility 'x86_64 aarch64 armv7l'
+  compatibility 'all'
   source_url 'https://gitlab.gnome.org/GNOME/evolution-data-server.git'
   git_hashtag @_ver
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/evolution_data_server/3.40.3_armv7l/evolution_data_server-3.40.3-chromeos-armv7l.tpxz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/evolution_data_server/3.40.3_armv7l/evolution_data_server-3.40.3-chromeos-armv7l.tpxz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/evolution_data_server/3.40.3_x86_64/evolution_data_server-3.40.3-chromeos-x86_64.tpxz'
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/evolution_data_server/3.46.0_i686/evolution_data_server-3.46.0-chromeos-i686.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/evolution_data_server/3.46.0_x86_64/evolution_data_server-3.46.0-chromeos-x86_64.tar.zst'
   })
   binary_sha256({
-    aarch64: '8ade9630fc18351495084f5e0ae3ad98fbbcd3e50f937d795d8dc2a1c4c36158',
-     armv7l: '8ade9630fc18351495084f5e0ae3ad98fbbcd3e50f937d795d8dc2a1c4c36158',
-     x86_64: '9620cad89b0ed797c84b4d7b85b977883678669963c724a679a09c681ef3934f'
+       i686: '303248f7eb4e050cbf846caaf9f23b04c19df4fa624ee90db6c799f891175271',
+     x86_64: '57189bbc3c4fa355db5d3e64dc89521f91e7366eadb251b993bd963ec573d1d3'
```

Comment:
Missing arm build?

Reply:
depends upon webkit2gtk_4_1, which isn't building for armv7l.

Reply:
I will build for armv7l if webkit2gtk_4_1 ends up building for armv7l...